kolibri/content/test/test_channel_import.py

```diff
@@ -319,45 +320,39 @@ def create_channel():
 
 DATA_PATH_TEMPLATE = os.path.join(os.path.dirname(__file__), '../fixtures/{name}_content_data.json')
 
-class NaiveImportTestCase(ContentNodeTestBase, TransactionTestCase):
-    """
-    Integration test for naive import - this is run using a TransactionTestCase,
+
+class ContentImportTestBase(TransactionTestCase):
```

Comment:
Because this is an instance of `TestCase`, I am pretty sure this is going to get executed in the test suite, but I'm not really sure what it is testing, as it has neither a name, nor a legacy schema associated with it.

This should inherit from object, and then you will have to mix in this, ContentNodeTestBase, and TransactionTestCase into NaiveContentImport.

Reply:
I think this should make everything else work out how I have intended, with regards to inheritance for prior versions also.